src/test/kotlin/com/openlattice/chronicle/candidates/CandidatesApiTests.kt

```diff
@@ -0,0 +1,36 @@
+package com.openlattice.chronicle.candidates
+
+import com.openlattice.chronicle.ChronicleServerTests
+import com.openlattice.chronicle.client.ChronicleClient
+import com.openlattice.chronicle.ids.IdConstants
+import org.junit.Test
+import java.lang.IllegalArgumentException
+
+class CandidatesApiTests : ChronicleServerTests() {
```

Comment:
These tests shouldn't pass unless I missed some validation in the Candidate class constructor. The ChronicleClient actually makes an actual REST request through the controller, which means that a non-200 response (i.e due to bad date) will actually get transformed into a RetrofitClientCallException not an IllegalArgumentException.

Reply:
Fixed.